### PR TITLE
Documentation | Added mssql-jdbc_auth Maven download link to README file

### PR DESCRIPTION
**Pull request by ulvii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ This driver is documented on [Microsoft's Documentation web site](https://docs.m
 For samples, please see the `src\sample` directory.
 
 ### Download the DLLs
-For some features (e.g. Integrated Authentication and Distributed Transactions), you may need to use the `sqljdbc_xa` and `mssql-jdbc_auth` DLLs. They can be downloaded from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?linkid=868287)
+For some features (e.g. Integrated Authentication and Distributed Transactions), you may need to use the `sqljdbc_xa` and `mssql-jdbc_auth-<version>.<arch>` DLLs. They can be downloaded from the [Microsoft Download Center](https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server). `mssql-jdbc_auth-<version>.<arch>` can also be downloaded from [Maven](https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc_auth).
```
Comment (peterbae) [inline README.md]: remove /en-us/

File: README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ This driver is documented on [Microsoft's Documentation web site](https://docs.m
 For samples, please see the `src\sample` directory.
 
 ### Download the DLLs
-For some features (e.g. Integrated Authentication and Distributed Transactions), you may need to use the `sqljdbc_xa` and `mssql-jdbc_auth` DLLs. They can be downloaded from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?linkid=868287)
+For some features (e.g. Integrated Authentication and Distributed Transactions), you may need to use the `sqljdbc_xa` and `mssql-jdbc_auth-<version>.<arch>` DLLs. They can be downloaded from the [Microsoft Download Center](https://docs.microsoft.com/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server). `mssql-jdbc_auth-<version>.<arch>` can also be downloaded from [Maven](https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc_auth).
```
Comment (lilgreenbird) [inline README.md]: should we mention AEv2 here?

Comment (ulvii) [reply README.md]: No need to mentioned all features here, but [this page ](https://docs.microsoft.com/en-us/sql/connect/jdbc/always-encrypted-with-secure-enclaves?view=sql-server-ver15)should be updated to mentioned DLLs.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [8.2.1] HotFix & Stable Release
+### Added
+- Added new Azure Key Vault endpoints to the list of trusted endpoints [#1264](https://github.com/microsoft/mssql-jdbc/pull/1264)
```
Comment (lilgreenbird) [inline CHANGELOG.md]: do we remove this now?

**Pull request closed without merging**